plugins/catalog-backend/migrations/20201005122705_add_entity_full_name.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// @ts-check
+
+/**
+ * @param {import('knex')} knex
+ */
+exports.up = async function up(knex) {
+  await knex.schema.alterTable('entities', table => {
+    table.text('full_name').nullable();
+  });
+
+  await knex('entities').update({
+    full_name: knex.raw(
+      "LOWER(kind) || ':' || COALESCE(LOWER(namespace), 'default') || '/' || LOWER(name)",
```

Comment:
`LOWER(NULL)` is fine? Figured `LOWER(COALESCE(namespace, 'default'))` might be safer

Reply:
It is, it returns null as far as I understood. And didn't coalesce due to cross-db oddness :/

Reply:
In general null isn't like null in other languages, it isn't an abnormal value but an absence of a value which makes operations not be performed at all (ie leading again to absence being returned) because there is nothing to apply on

Reply:
fix'd anyway because it does nice